From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <sattard@slack-corp.com>
Date: Mon, 15 Jul 2019 17:45:02 -0700
Subject: chore: read _noBrowserGlobals from global not config

This is used so that we can modify the flag at runtime where
config can only be set at compile time.

diff --git a/lib/internal/bootstrap/node.js b/lib/internal/bootstrap/node.js
index e1f70addc28e4fe31d3a7089ecec3b5874dde75a..659145e31bbfb6ce782e8dcad452b59202df7573 100644
--- a/lib/internal/bootstrap/node.js
+++ b/lib/internal/bootstrap/node.js
@@ -118,7 +118,7 @@ const {
   queueMicrotask
 } = require('internal/process/task_queues');
 
-if (!config.noBrowserGlobals) {
+if (!global._noBrowserGlobals) {
   // Override global console from the one provided by the VM
   // to the one implemented by Node.js
   // https://console.spec.whatwg.org/#console-namespace
